os_stub/mbedtlslib/CMakeLists.txt

```diff
@@ -5,6 +5,40 @@ else()
     set(LIB_INSTALL_DIR lib)
 endif()
 
+# Create a symbolic link from ${base_name} in the binary directory
```

Comment:
Question: Is this mandatory for the upgrade?
If not, I recommend to handle it in another PR, because the grand rule is: one PR handles one issue.

There is no problem to add new feature. We just need to handle them one by one.

Reply:
> Question: Is this mandatory for the upgrade?

No, In recent mbedtls library/CMakeLists.txt refers a function link_to_source() that is implemented in top level cmake file. I can add a link_to_source() in os_stub/mbedtlslib/CMakeLists.txt that just returns without doing the source file linking.

> There is no problem to add new feature. We just need to handle them one by one.
Sure. This can be handled later if required.

Reply:
Removed link_to_source implementation.